src/app/reporting/Engine.h

```diff
@@ -119,6 +127,8 @@ class Engine
      *
      */
     uint32_t mCurReadHandlerIdx = 0;
+
+    ClusterInfo * mpChangePathList = nullptr;
```

Comment:
```suggestion
    ClusterInfo * mpGlobalChangeList = nullptr;
```

You have a mixture of methods that have the term 'dirty' in them, and a mixture that use 'Change' as the term. Please pick one or the other (my vote: dirty, since it has a basis in caches and is well understood).

Reply:
Just update it with mpGlobalDirtyList